app/controls/open-browser.js

```diff
@@ -1,23 +1,24 @@
 import ipc from 'electron-safe-ipc/host'
 import {apiAddrToUrl} from './utils'
-import {getIPFS} from './../init'
+import {logger, getIPFS} from './../init'
```

Comment:
can we get the `logger` coming from a `config.js` file, along with other default values?